prelude/bootstrap.js

```diff
@@ -259,14 +265,7 @@ function copyFolderRecursiveSync(source, target) {
 function createDirRecursively(dir) {
   if (!fs.existsSync(dir)) {
     createDirRecursively(path.join(dir, '..'));
```

Comment:
Cannot this also create the same error?

Reply:
Even if I'm more familiar with codebase I couldn't tell you what else could go wrong, BTW let's see how tests goes